core/src/main/scripts/importer/validateStudies.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python2.7
+
+"""
+Copyright (c) 2018 The Hyve B.V.
+This code is licensed under the GNU Affero General Public License (AGPL),
+version 3, or (at your option) any later version.
+"""
+
+import argparse
+import os
+import logging
+import traceback
+
+import validateData
+
+
+def main(args):
+    """Process arguments and run validation"""
+
+    # Set up logger, stdout from validation will be written to a log file
+    logging.basicConfig(level=logging.DEBUG,
+                        filename='log-validate-studies.txt',
+                        filemode='w')
```

Comment:
Let's not write the log file, and throw away the standard out. I checked the log files and the contents are not very useful compared to what is in HTML reports.

Reply:
please write log file to temp dir

Reply:
After discussion with @pieterlukasse we decided to keep the log file, but that it has to be written to the HTML-folder (if provided) otherwise to a created tmp folder.